homeassistant/components/system_bridge/services.yaml

```diff
@@ -70,3 +70,28 @@ send_text:
       example: "Hello world"
       selector:
         text:
+power_command:
+  name: Power Command
+  description: Sends a power command to the system.
```

Comment:
Please get your branch up to date, as all names and descriptions are moved to `strings.json`